tests/test_definitions/column_map_expectations/expect_column_values_to_be_of_type.json

```diff
@@ -33,6 +33,12 @@
         },
         "redshift": {
           "x": "INTEGER"
+        },
+        "bigquery": {
```

Comment:
We shouldn't rely on these json files any more. I've been trying to just leave behind one test in these (the existence of these tests is used by some test about rendered content where it counts the number of expectations, and I haven't had bandwidth to replace that). But I would definitely not add to this file.